### PR TITLE
MIGRATION app csproj to net5.0, updated all project references

### PR DESCRIPTION
**Pull request by digitalsigi**
CHANGE SQL Provider uses now UseIdentityColumn
CHANGE some fluentassertions seem to be no longer there, changed the respective sources

### CODE REVIEW

**Comment by digitalsigi** [comment]
I did not update all version strings, because I had no glue what your numbering schema would be.  
However I updated those for WorkflowCore, workflowcore.persistence.entityframework and workflowcore.persistence.sqlserver

File: src/providers/WorkflowCore.Persistence.PostgreSQL/WorkflowCore.Persistence.PostgreSQL.csproj [hunk 1]
```diff
@@ -27,12 +27,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Npgsql" Version="4.1.3.1" />
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.2" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="3.1.2">
+    <PackageReference Include="Npgsql" Version="4.1.5" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="3.1.4" />
```
Comment (evolvedlight) [inline src/providers/WorkflowCore.Persistence.PostgreSQL/WorkflowCore.Persistence.PostgreSQL.csproj:31]: can you bump these to v5 please?
https://www.nuget.org/packages/Npgsql.EntityFrameworkCore.PostgreSQL/
https://www.nuget.org/packages/Npgsql/

**Review by danielgerlag — COMMENTED** [review]
As this is a library, I would prefer not to force all consumers of the library to upgrade in-step.  I would prefer to target the broadest possible set of framework versions out there, unless there is a specific feature in the newer version that we require.

**Comment by evolvedlight** [comment]
@danielgerlag EF Core 5.0 target .Net Standard 2.1, so isn't compatible with .Net Framework anymore. 
But if you don't upgrade to EF Core 5.0, then we can't use this library in a .Net 5.0/Ef Core 5.0 application, as it's no longer compatible. So it seems for workflow core you'll need to choose between either supporting ef core 5.0, or supporting .Net Framework.
For us this is a problem - we'd like to use EF Core 5.0 features in an application that also has workflow core dependency - so at the moment workflow core is stopping us upgrading.

**Comment by digitalsigi** [comment]
@danielgerlag Same here, had the requirement to upgrade to .net50 so I updated WorkflowCore. I am aware that this might result in some discrepances, but saw no other choice.

**Comment by digitalsigi** [comment]
@evolvedlight @danielgerlag I will investigate whether a multi-target approach is possible. But that may take time as I have no experience in this corner.

**Comment by danielgerlag** [comment]
In this case, we should only upgrade the libraries that depend  on EF core, rather than everything.

**Comment by evolvedlight** [comment]
@danielgerlag to be clear, does that mean you will accept a PR that means that WorkflowCore looses support for .Net Framework?

**Comment by danielgerlag** [comment]
@evolvedlight no, not for the core library.  We can update the EF Core libraries but keep all others targeting .Net Standard 2.0

**Comment by evolvedlight** [comment]
@danielgerlag Sorry, of course. I'll make a PR for this - would you like me to bump the major version of the libraries too?

**Comment by glucaci** [comment]
I would also don't do any change to core libraries, it should stay as low as possible with the target framework, than you don't have to change the version.

**Comment by digitalsigi** [comment]
Fork deleted. Thx to all.

**Pull request closed without merging**